integration_tests/06_docker_network_ping-pong.sh

```diff
@@ -10,11 +10,11 @@ docker tag $ARTIFACT testplan:network
 
 pushd $TEMPDIR
 testground healthcheck --runner local:docker --fix
-testground run single --runner local:docker --builder docker:go --use-build testplan:network --instances 2 --plan network --testcase ping-pong --collect | tee run.out
-RUNID=$(awk '/finished run with ID/ { print $9 }' run.out)
+testground run single --runner local:docker --builder docker:go --use-build testplan:network --instances 2 --plan network --testcase ping-pong --collect | tee stdout.out
+RUNID=$(awk '/finished run with ID/ { print $9 }' stdout.out)
 echo "checking run $RUNID"
 file $RUNID.tgz
-unzip $RUNID.tgz
+tar -xzvvf $RUNID.tgz
```

Comment:
nit: double `v` (unless it's intentional?)

Reply:
Good question, this is how I have remembered that command, let me double check it.

Reply:
So this was intentional (I always use that command). There is difference between `-v` and `-vv` - the latter shows also unix owner and group:

```
➜  git:(e473f629) ✗ tar -xzvf 8f0b93c63009.tgz
x 8f0b93c63009/.
x 8f0b93c63009/single
x 8f0b93c63009/single/0
x 8f0b93c63009/single/0/diagnostics.out
x 8f0b93c63009/single/0/results.out
x 8f0b93c63009/single/0/run.err
x 8f0b93c63009/single/0/run.out
x 8f0b93c63009/single/1
x 8f0b93c63009/single/1/diagnostics.out
x 8f0b93c63009/single/1/results.out
x 8f0b93c63009/single/1/run.err
x 8f0b93c63009/single/1/run.out
➜  git:(e473f629) ✗ tar -xzvv
➜  git:(e473f629) ✗ rm -rf 8f0b93c63009
➜  git:(e473f629) ✗ tar -xzvvf 8f0b93c63009.tgz
x drwxr-xr-x  0 nonsense staff       0 Jun 15 16:34 8f0b93c63009/.
x drwxr-xr-x  0 nonsense staff       0 Jun 15 16:34 8f0b93c63009/single
x drwxr-xr-x  0 nonsense staff       0 Jun 15 16:34 8f0b93c63009/single/0
x -rw-r--r--  0 nonsense staff    3473 Jun 15 16:34 8f0b93c63009/single/0/diagnostics.out
x -rw-r--r--  0 nonsense staff       0 Jun 15 16:34 8f0b93c63009/single/0/results.out
x -rw-r--r--  0 nonsense staff       0 Jun 15 16:34 8f0b93c63009/single/0/run.err
x -rw-r--r--  0 nonsense staff    3993 Jun 15 16:34 8f0b93c63009/single/0/run.out
x drwxr-xr-x  0 nonsense staff       0 Jun 15 16:34 8f0b93c63009/single/1
x -rw-r--r--  0 nonsense staff       0 Jun 15 16:34 8f0b93c63009/single/1/diagnostics.out
x -rw-r--r--  0 nonsense staff       0 Jun 15 16:34 8f0b93c63009/single/1/results.out
x -rw-r--r--  0 nonsense staff       0 Jun 15 16:34 8f0b93c63009/single/1/run.err
x -rw-r--r--  0 nonsense staff    3992 Jun 15 16:34 8f0b93c63009/single/1/run.out
```